wazo_debug/http_request_duration.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import argparse
+import subprocess
+
+from cliff.command import Command
+
+
+class HTTPRequestDurationCommand(Command):
+    "Display all HTTP requests sorted by response duration"
+
+    def get_parser(self, program_name):
+        parser = argparse.ArgumentParser(
+            prog=program_name,
+            description='Display HTTP requests sorted by response duration. The first column is the request duration in seconds.',
+        )
+        parser.add_argument(
+            '--access-file',
+            action='store',
+            required=False,
+            default='/var/log/nginx/wazo.access.log',
+            help='The Nginx access file to analyze',
+        )
+        return parser
+
+    def take_action(self, parsed_args):
+        print(f'Analyzing log file {parsed_args.access_file}')
+        command = (
```

Comment:
I would like to allow gz files to be checked
```diff
diff --git a/wazo_debug/http_request_duration.py b/wazo_debug/http_request_duration.py
index 408aa77..6e548fe 100644
--- a/wazo_debug/http_request_duration.py
+++ b/wazo_debug/http_request_duration.py
@@ -27,8 +27,8 @@ class HTTPRequestDurationCommand(Command):
     def take_action(self, parsed_args):
         print(f'Analyzing log file {parsed_args.access_file}')
         command = (
-            '''awk -F' ' '{ print $(NF-1) " " $0 }' '''
-            + parsed_args.access_file
+            '''zgrep '' ''' + parsed_args.access_file
+            + '''| awk -F' ' '{ print $(NF-1) " " $0 }' '''
             + ''' | grep -v websocket | grep -v asterisk/ws | sort -n | less +G'''
         )
         try:
```

Reply:
The `zgrep '' <filename>` behaves like a `cat` or `zcat` without having to check the file type

Reply:
Great idea, thanks :)
I used `zcat -f` instead of `zgrep`

Reply:
Cool, I didn't know about `zcat -f` thanks